nvm.sh

```diff
@@ -3132,6 +3151,7 @@ nvm() {
       local ALIAS
       local NVM_UPGRADE_NPM
       NVM_UPGRADE_NPM=0
+      local NVM_WRITE_TO_NVMRC
```

Comment:
```suggestion
      local NVM_WRITE_TO_NVMRC
      NVM_WRITE_TO_NVMRC=0
```
to make sure it's set in every code path

Reply:
I took away that declaration so that the `NVM_WRITE_TO_NVMRC` from `nvm install` stays set when `nvm use` gets called, which I thought was better than appending `--save` to each of the `nvm use` calls within the `nvm install` block

Reply:
Oh never mind, that's not what you were referring to - can do although it shouldn't make a difference since I'm just using a default arg when checking (`[ "${NVM_WRITE_TO_NVMRC:-0}" -eq 1 ]`)